contents/blog/ab-testing-mistakes.md

```diff
@@ -0,0 +1,172 @@
+
+To avoid this problem, you should first test your experiment with a small rollout (e.g., 5% of users) for a few days. Once you're confident everything works correctly, you can start the experiment with the remaining users.
+
+Here is a list of what to check during your test rollout:
+* No increase in crashes or other errors.
+* Logging is working correctly.
+* Use [session replays](/session-replay) to ensure your app is behaving as expected.  
+* Users are assigned to the control and test groups in the ratio you are expecting (e.g., 50/50).
+
+## 5. Neglecting counter metrics
+
+Counter metrics measure unintended negative side-effects in your experiments. If you do not monitor them, you may accidentally roll out changes that result in a worse user experience. 
+
+For example, say you're testing a change to your sign-up page. While the number of sign-ups may increase, you notice that time spent in your app decreases. In this case, it may indicate that your new sign-up page is misleading users about what your app does, resulting in more sign-ups but also more churn.
+
+## 6. Not accounting for seasonality
+
+Seasonal periods can cause significant changes in user behavior. People might be taking time off or focusing on different tasks. In a B2B context, seasonality can also affect the decision-making processes of businesses. For example, companies often have specific times of the year when they allocate budgets, review contracts, or make purchasing decisions. Conducting A/B tests during these periods can result in biased outcomes that may not represent typical user behavior.
+
+Here is a list of seasonality periods to be mindful of:
+
+* Holiday seasons: e.g., Christmas and New Year's, July and August summer vacations.
+* Fiscal year-end: Companies typically review budgeting decisions and contract renewals during this period.
+* Seasonal sales cycles: Some industries, like retail or agriculture, have seasonal sales cycles. Be aware of these cycles if your target customers operate in such industries.
+
+## 7. Testing an unclear hypothesis
+
+Having a clear definition of what you are testing and why you are testing it will help you determine which metrics to measure and analyze. To understand this better, let's look at an example of a bad hypothesis:
+
+> Bad Hypothesis: Changing the color of the "Proceed to checkout" button will increase purchases.
+
+This is bad since it's unclear why we're testing this change and why we expect it to increase purchases. As a result, it's not clear what we need to measure in this experiment. Do we only need to count button clicks? Or is there something else we need to measure?
+
+Here's an improved version:
+
+> Good hypothesis: User research showed that users are unsure of how to proceed to the checkout page. Changing the button's color will lead to more users noticing it and thus more people will proceed to the checkout page. This will then lead to more purchases.
```

Comment:
```suggestion
> **Good hypothesis:** User research showed that users are unsure of how to proceed to the checkout page. Changing the button's color will lead to more users noticing it and thus more people will proceed to the checkout page. This will then lead to more purchases.
```